core/tests/test_apps.nem.mosaic.py

```diff
@@ -22,36 +29,36 @@ def test_get_mosaic_definition(self):
         self.assertEqual(m, None)
 
         m = get_mosaic_definition("pacnem", "cheese", 104)
-        self.assertEqual(m["name"], "PacNEM Score Tokens")
-        self.assertEqual(m["ticker"], " PAC:CHS")
-        self.assertEqual(m["fee"], 100)
+        self.assertEqual(m.name, "PacNEM Score Tokens")
+        self.assertEqual(m.ticker, " PAC:CHS")
+        self.assertEqual(m.fee, 100)
 
     def test_mosaic_canonicalization(self):
-        a = NEMMosaic()
+        a = NEMMosaic(**mosaic_kwargs)
```

Comment:
this should instead be
```
a = NEMMosaic(
    namespace="abc",
    quantity=3,
    mosaic="mosaic",
)
```

Reply:
Done in 329a038 with a small helper function to improve readability